gstreamer-video/src/auto/video_decoder.rs

```diff
@@ -0,0 +1,295 @@
+// This file was generated by gir (0fe730d) from gir-files (???)
+// DO NOT EDIT
+
+use VideoCodecFrame;
+use VideoCodecState;
+use VideoFormat;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use gst;
+use gst_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct VideoDecoder(Object<ffi::GstVideoDecoder>): [
+        gst::Element => gst_ffi::GstElement,
+        gst::Object => gst_ffi::GstObject,
+    ];
+
+    match fn {
+        get_type => || ffi::gst_video_decoder_get_type(),
+    }
+}
+
+unsafe impl Send for VideoDecoder {}
+unsafe impl Sync for VideoDecoder {}
+
+pub trait VideoDecoderExt {
+    fn add_to_frame(&self, n_bytes: i32);
+
+    fn allocate_output_buffer(&self) -> Option<gst::Buffer>;
+
+    fn allocate_output_frame(&self, frame: &VideoCodecFrame) -> gst::FlowReturn;
+
+    //#[cfg(any(feature = "v1_12", feature = "dox"))]
+    //fn allocate_output_frame_with_params(&self, frame: &VideoCodecFrame, params: /*Ignored*/&mut gst::BufferPoolAcquireParams) -> gst::FlowReturn;
+
+    fn drop_frame(&self, frame: &VideoCodecFrame) -> gst::FlowReturn;
+
+    fn finish_frame(&self, frame: &VideoCodecFrame) -> gst::FlowReturn;
+
+    //fn get_allocator(&self, allocator: /*Ignored*/gst::Allocator, params: /*Ignored*/gst::AllocationParams);
+
+    //fn get_buffer_pool(&self) -> /*Ignored*/Option<gst::BufferPool>;
+
+    fn get_estimate_rate(&self) -> i32;
+
+    fn get_frame(&self, frame_number: i32) -> Option<VideoCodecFrame>;
+
+    fn get_frames(&self) -> Vec<VideoCodecFrame>;
+
+    fn get_latency(&self) -> (gst::ClockTime, gst::ClockTime);
+
+    fn get_max_decode_time(&self, frame: &VideoCodecFrame) -> gst::ClockTimeDiff;
+
+    fn get_max_errors(&self) -> i32;
+
+    fn get_needs_format(&self) -> bool;
+
+    fn get_oldest_frame(&self) -> Option<VideoCodecFrame>;
+
+    fn get_output_state(&self) -> Option<VideoCodecState>;
+
+    fn get_packetized(&self) -> bool;
+
+    fn get_pending_frame_size(&self) -> usize;
+
+    fn get_qos_proportion(&self) -> f64;
+
+    fn have_frame(&self) -> gst::FlowReturn;
+
+    //fn merge_tags<'a, P: Into<Option<&'a /*Ignored*/gst::TagList>>>(&self, tags: P, mode: /*Ignored*/gst::TagMergeMode);
```

Comment:
You get these generated if you add taglist and tag merge mode as manual into the toml file. See how it's done for taglist (or caps, structure, etc) in the other toml files.

Reply:
Done.